### PR TITLE
Rails 7

### PR DESCRIPTION
**Pull request by yosiat**
Hi!

Rails 7 alpha 2 is out, and I am currently working on getting my application to pass tests with it.

Tested locally with rails 7 alpha 2 and all tests pass, there is only one deprecation notice from test helper on rails 7.1 -
```
DEPRECATION WARNING: ActiveRecord::Base.default_timezone= is deprecated and will be removed in Rails 7.1.
Use `ActiveRecord.default_timezone=` instead.
 (called from <top (required)> at /Users/yosi/code/where_exists/test/test_helper.rb:8)
```

### CODE REVIEW

**Comment by EugZol** [comment]
@yosiat Hi, thanks!

Does the new suggested `ActiveRecord.default_timezone=` work in Rails 5.2? If so, we can fix it and drop Rails 4.x support.

**Comment by yosiat** [comment]
@EugZol all tests are passing with rails 5.2 :)

If you want, I can submit PR for adding CI with Github Actions.

**Comment by EugZol** [comment]
@yosiat That would be very much appreciated. I admit I worked only with GitLab CI. Would be interesting to integrate GitLab Actions here.

Regarding this PR:

* please change minimum Rails version to '5.2'
* bump gem version to '2.0.0'
* change minimum Rails version in the README

**Comment by yosiat** [comment]
Hi @EugZol 

Added this to PR - Github Actions (see results here - https://github.com/yosiat/where_exists/actions/runs/1534771257)

* please change minimum Rails version to '5.2' - ✅ 
* bump gem version to '2.0.0' - ✅ 
* change minimum Rails version in the README - ✅ 
* remove/update deprecated code - I am not familiar with code the base, If you can give me guidelines to which code I should update it will help me.

File: Appraisals [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+appraise '6.0' do
+ gem 'rails', '~> 6.0'
+end
+
+appraise '6.1' do
+ gem 'rails', '~> 6.1'
+end
+
+appraise '7' do
+  gem 'rails', '7.0.0alpha2'
```
Comment (yosiat) [inline Appraisals:15]: @EugZol once Rails 7 will be official released, I'll do a PR here to update the version number

**Comment by yosiat** [comment]
@EugZol hey...
I added CI, updated to test with Rails 7.. what is more left for this PR to get merged?

**Comment by EugZol** [comment]
@yosiat Please remove `ActiveRecord::Base.default_timezone=` from the code – which you mentioned in the start of this thread. I will deal with this PR on the weekends. Sorry for the delays.

**Comment by yosiat** [comment]
@EugZol thanks for merge !

And delays - are ok and acceptable, it's open source project :)

**Comment by EugZol** [comment]
@yosiat Pushed 2.0.0. Thanks for the interesting PR: enjoyed the workflows :)

**Comment by yosiat** [comment]
Thanks! going to update now :)

About workflows - yes liked the outcome as well, it's simpler than panko workflow I have.. I'll replace in the near future to have same setup as here :)

**Pull request merged**